recipes/mss/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set version = "1.1.0" %}
+
+
+package:
+  name: mss
+  version: {{ version }}
+
+source:
+    git_url: https://bitbucket.org/wxmetvis/mss.git
+    git_rev: 1.1.0
+
+build:
+    number: 0
+    script: python setup.py install --record record.txt
+    entry_points:
+      - mss = msui.mss_pyui:main
+
+requirements:
+  build:
+    - basemap ==1.0.7
+    - chameleon >=2.24
+    - curl ==7.26
+    - cycler ==0.10.0
+    - execnet ==1.2.0
```

Comment:
You are right, currently trying ;)
 At the moment I have a simpler test for that curl issue it would be much easier.

The pinning was done because these versions were verified that the whole package besides the ssl issue runs on windows and linux.

There were some packages which created issues, e.g. basemap-1.0.8  on client and server but some only on server and only in TLS (ssl) setup